### DISCUSSION TITLE
Reboot/Shutdown seem to be broken after latest update...

### DISCUSSION THREAD

**Discussion by cmh714** [rern/rAudio#1447] [Bug Reports]
Also doesnt work from the commandline for me. I have even reflashed several times and still broken. I have updated via the ui and also pacman....unsure what up atm.

TY!!

**Comment by cmh714** [comment]
to add.....if I ONLY update with pacman, reboot/shutdown still works. If I ONLY update via the Addons page, reboot/shutdown still work.

if I run both, then it seems to break.

**Comment by klassikmann** [comment]
@cmh714 
I made similar experiences:
- https://github.com/rern/rAudio/discussions/1438
- https://github.com/rern/rAudio/discussions/1444

**Comment by klassikmann** [comment]
@cmh714 
I'm testing a system based on the (older) i20230828 image. Up to now it's working (reboot/shutdown, pacman -Syu, Kid3 tag editor). I'll proceed the tests, then I'll report here.

**Comment by klassikmann** [comment]
@cmh714 
@R101G
@rern 
> I'm testing a system based on the (older) i20230828 image. Up to now it's working (reboot/shutdown, pacman -Syu, Kid3 tag editor). I'll proceed the tests, then I'll report here.

All the problems I faced with an installation based on the newest image i20230918 (reboot/shutdown, pacman -Syu, others) and a DIY-system according https://github.com/rern/rOS (Kid3 tag editor) are gone using the predecessor image (i20230828). 

My hypothesis:
Newest image (i20230918) and actual rOS (DIY rAudio) are buggy (IMHO).

**Reply by cmh714** [reply]
TY Sir!! Working on that now! Appreciate your effort!

**Reply by cmh714** [reply]
REVISIT! So while this "seemed" to work, I was unable to rebuild my library. It was trying for over 12 hours! :)

It also doesnt like restoring a backup properly. The restored mount points were unable to mount. So more messing around today.....

Thanks!!!

**Reply by klassikmann** [reply]
@cmh714 
1. I made a full rebuild of the library: 8 minutes.
2. Mount point of the NAS o.k., access to the files on the NAS o.k.
3. In case of relevance: Because of my large library I set the parameter max_output_buffer_size in the player menu to 16384 a long time ago already.

Could your backup.gz you used for restoring be corrupt? Please inspect the backup file for some irregularities, e.g. regarding mount points in data/config/etc/fstab.

btw. It looks as rern is preparing a new release (many changes in the UPDATE branch during the last couple of hours).

EDIT: A new mpd version (0.23.14) has been released 3 days ago and is present if you made a pacman -Syu. No problems here.

**Reply by cmh714** [reply]
TY Again! Nothing worked for me yesterday. I have over 172,000 Flac files (6TB) so dont think there are too many libraries bigger :)
My settings are always at 32768, so thats not it.....at least I dont believe so. I will keep trying, thankfully I have an Aurender N10.... 

but again still love and need Rune on Pi!!

**Reply by klassikmann** [reply]
@cmh714 

fyi 
1.  At the time being I have no  problems with my rAudio installation with the exception of two minor one, where I didn't find spare time for analysis up to now. But my configuration is very slim without using any of the optional "Features" besides "Lyrics",
2. rAudio on a RPi 3B is my sole audio player. I do not use a DAC but a "Hifiberry Digi+ Pro" to transport the sound  "bit-perfect" via S/PDIF up to 24bit/192kHz to my active studio monitors (Genelec 8351).

**Comment by klassikmann** [comment]
@cmh714 
Tomorrow I'll have a look into library rebuilding. My backup restored without problems (Data + Mountpoints).

**Reply by cmh714** [reply]
Good to know. I am still jacking around as well! Will report anything useful.

**Comment by cmh714** [comment]
I noticed that the mount points for NAS werent created during restore so I manually created them. That "seemed" to work but I cannot get to the files, so it really did not work. Think I may give up for the time being as I have an expensive player/streamer to use.....but I still love Raudio!

**Comment by cmh714** [comment]
So still trying to slog through this :) and have gotten closer but still issues.
1. Rebuild entire DB doesnt seem to work and "rebuilds" forever.
2. Rebuild Only Changed Files "seems" to work, but I cant get all of my files indexed. I had 172k before and now only have 117k....havent yet figured out what may be missing....

otherwise good....
